docs/docs/getting-started/logging-first-model/notebooks/index.mdx

```diff
@@ -0,0 +1,19 @@
+---
+sidebar_position: 7
+---
+
+# Logging Your First MLflow Model Notebook
+
+If you would like to view the full notebook:
+
+<a class="button button--primary" href="/notebooks/logging-first-model">
```

Comment:
When the notebook migration script is done, it will create a corresponding `.mdx` file in the same directory as the source `.ipynb` files. In this case, `docs/getting-started/logging-first-model/notebooks/logging-first-model.mdx` will be created

```suggestion
<a class="button button--primary" href="./logging-first-model">
```